consensus/cw_rating_calculator/cw_rating_dfs_impl.h

```diff
@@ -5,14 +5,18 @@
  * Refer to the LICENSE file for licensing information
  */
 
-#ifndef __COMMON_CONCENSUS_CW_RATING_CALCULATOR_CW_RATING_TOPOLOGICAL_SORT_IMPL_H__
-#define __COMMON_CONCENSUS_CW_RATING_CALCULATOR_CW_RATING_TOPOLOGICAL_SORT_IMPL_H__
-
-#include "consensus/cw_rating_calculator/cw_rating_calculator.h"
+#ifndef __CONSENSUS_CW_RATING_CALCULATOR_CW_RATING_TOPOLOGICAL_SORT_IMPL_H__
```

Comment:
should be DFS rather than topological_sort, forgot that last time

Reply:
Will fix